### ISSUE TITLE
poweredby link doesnt conform to accessibility standards

### ISSUE THREAD

**Issue by davidfurlong** [algolia/react-instantsearch#209]
PoweredBy Widget:

Using lighthouse

Elements Have Discernable Names
Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content.

Links do not have a discernable name.
Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users. Learn more.
View failing elements

`<a href="https://www.algolia.com/?utm_source=react-instantsearch&amp;utm_medium=website&amp;utm_content=staging.deedmob.com&amp;utm_campaign=poweredby" target="_blank" class="ais-PoweredBy__algoliaLink">`

**Comment by Haroenv** [comment]
Is adding a `title` enough according to Lighthouse, or do they prefer `aria-label`?

**Comment by davidfurlong** [comment]
@Haroenv the title attribute is not read by screenreaders by default. Given the ambigious content of the link I would recommend the  `aria-label` property https://dev.opera.com/articles/ux-accessibility-aria-label/#accessible-name-calculation

**Comment by Haroenv** [comment]
That's what I thought. Let's get this done. If you're quicker, it should be translation key like in other components, with a default value of "search by Algolia, learn more" I think. Feel free to open a PR 😊
